docs/environments.md

```diff
@@ -1,103 +1,57 @@
-You are now connected to database "thoth" as user "postgres".
-thoth=# CREATE EXTENSION amcheck;
-CREATE EXTENSION
-```
-
-It can be verified if installed via `\dx`
-
-```bash
-thoth=# \dx
-                                    List of installed extensions
-        Name        | Version |   Schema   |                        Description
---------------------+---------+------------+-----------------------------------------------------------
-amcheck            | 1.0     | public     | functions for verifying relation integrity
-pg_stat_statements | 1.6     | public     | track execution statistics of all SQL statements executed
-plpgsql            | 1.0     | pg_catalog | PL/pgSQL procedural language
-(3 rows)
-```
 
-**Note**: `amcheck functions may be used only by superusers.` we currectly have made our user the **superuser**, based on your setup either do the same or create different users, assign them priviledges, and set them up in pgbouncer.
+## Stage Environments
 
-Links:
+This environment is behind **Red Hat VPN**, it would require VPN for access.<br>
+The stage environment is deployed on the OCP cluster named **ocp4** hosted by PSI.<br>
+The access to this cluster is only available for [thoth](https://rover.redhat.com/groups/group/aicoe-thoth) team only,which is configured directly via ldap groups.<br>
+The deployment on this cluster is set via ArgoCD and also set on self-healing and pruning of resources.<br>
+It uses an ephemeral PostgreSQL deployment,and depends on an external Kafka Broker deployment.
 
-- [Configure pg_stat_statements extension](https://github.com/thoth-station/thoth-application/issues/726)
-- [Setup postgresql admin](https://github.com/thoth-station/thoth-application/issues/725)
-- [related issue](https://github.com/thoth-station/storages/issues/2250)
-- [documentation/amcheck-next](https://access.crunchydata.com/documentation/amcheck-next/1.5/)
+Server: [ocp4-psi](https://ocp4.psi.redhat.com/)<br>
+Request access: Contact the Manager to be included in the [group](https://rover.redhat.com/groups/group/aicoe-thoth).
 
 ## Test Environments
 
-There might be multiple and personal test environment, but there is also one provided to the whole team to hack around. This test environment can be broken at any time, ArgoCD is not self-healing it or pruning unrelated resources. The test environment consists of one OpenShift project, uses an ephemeral PostgreSQL deployment inside of that project, and depends on an external Kafka Broker deployment.
-
+The Test environments are volatail and can be multiple at a give point of time, however devs can deploy there own if needed on the quicklab cluster (read here for more). The hosted test environment by team can be broken at any time, and ArgoCD is not set on self-healing or pruning of resources.<br> The test environment consists of one OpenShift project, uses an ephemeral PostgreSQL deployment inside of that project, and depends on an external Kafka Broker deployment.<br>
```

Comment:
```suggestion
The Test environments are volatile and can be multiple at any given point in time, however devs can deploy their own if needed on the quicklab cluster (read here for more). The hosted test environment by team can be broken at any time, and ArgoCD is not set on self-healing or pruning of resources.<br>
The test environment consists of one OpenShift project, uses an ephemeral PostgreSQL deployment inside of that project, and depends on an external Kafka Broker deployment.<br>
```
spell.

Reply:
Also, what exactly does it mean that test environments can be multiple at any given in time ? Shouldn't we simply document what exists ?